autorest/azure/persist_test.go

```diff
@@ -90,8 +98,12 @@ func TestSaveToken(t *testing.T) {
 		t.Fatalf("azure: unexpected error when creating temp file: %v", err)
 	}
 	defer os.Remove(f.Name())
+	f.Close()
 
 	mode := os.ModePerm & 0642
+	if runtime.GOOS == "windows" {
```

Comment:
yeah, that makes more sense... what would we be testing anyway...
I'll change that